part-configuration.tex

```diff
@@ -408,6 +408,14 @@ \subsection{LaTeX templates}
 
 Same as the last subsection.
 
+To add a file to the latex template upload the file to the Files directory.  For example, "FL\_Logo\_icon\_250x250.png", then add something like the following to the template.

```

Comment:
I think this needs a bit of clarification by referencing menus that need traversing and maybe a screenshot of the page where the file can be uploaded?

Reply:
Agreed working on it.

Reply:
Done.